### ISSUE TITLE
Make quickstarts quicker

### ISSUE THREAD

**Issue by johananl** [kinvolk/lokomotive#273]
The goal of a quickstart guide is **let the user have something working ASAP**.

Non-goals of a quickstart guide:

- Explain things which aren't required to get a minimal setup working.
- Provide a production-ready setup (unless we can provide a production-ready cluster using mainly default options, which is unlikely IMO).
- List all available options.
- Discuss edge cases.

Our quickstart guides in their current form could use some improvement. They don't always include all the necessary information in a clear and concise way, and often include extra information which "clutters" the guide and makes it harder to follow.

Therefore, I suggest we refactor our quickstart guides. I propose we use the following guidelines:

- Include **all the necessary information but no additional information**. Any text or code snippet that's required to get a working cluster should be included inline in the guide's **body**. Any text which isn't **directly related** to getting a minimal Lokomotive cluster working shouldn't exist in the guide.
- Link to other documents to provide more information on a topic, discuss edge cases or recommend what to do next after completing the guide. To re-emphasize, linking to other documents should only be done for content that isn't required to get a working cluster.
- Use concrete examples instead of placeholder text (see https://github.com/kinvolk/lokomotive/issues/244). Instead of writing `cluster_name = "<your_cluster_name_here>"`, write `cluster_name = "lokomotive-demo"`. Concrete examples make it easier to understand the **format** of values.
- When there are multiple options, pick a common one and focus on it. Don't try to cover all the available alternatives in a quickstart - that's the purpose of a **reference guide**.
- Minimize "running around" for the user. Include all the necessary information for a given step inline and in context. For example, rather than having the user scroll or click links to figure out what value to put in some config option, provide a copy-pasteable example which just works.

**Comment by johananl** [comment]
Some concrete examples which IMO can be improved:

https://github.com/kinvolk/lokomotive/blame/5e4f3c2884c1d018e29186d5aee68d4b51d1f653/docs/quickstarts/packet.md#L45-L46

>To install `lokoctl`, follow the instructions in the [lokoctl installation](../installer/lokoctl.md)
guide.

Since installing `lokoctl` isn't a *requirement* but rather an integral part of the quickstart, the required steps should be included inline and not linked to.

https://github.com/kinvolk/lokomotive/blame/5e4f3c2884c1d018e29186d5aee68d4b51d1f653/docs/quickstarts/packet.md#L50-L53

>It's better to start fresh in a new working directory, as the state of the cluster is stored in this
directory.

>This also makes the cleanup task easier.

I find this phrasing too elaborate. I would instead write this step as follows:

>Create a directory for your cluster configuration and navigate to it:

```console
mkdir -p lokomotive-infra/mycluster
cd lokomotive-infra/mycluster
```

https://github.com/kinvolk/lokomotive/blame/5e4f3c2884c1d018e29186d5aee68d4b51d1f653/docs/quickstarts/packet.md#L64

>* Log in to your Packet account and obtain the Project ID from the `Project Settings` tab.

Having a Packet project ID is a **requirement** for the guide. We shouldn't explain how to satisfy this requirement as it's out of scope.

https://github.com/kinvolk/lokomotive/blame/5e4f3c2884c1d018e29186d5aee68d4b51d1f653/docs/quickstarts/packet.md#L68-L70

```console
export PACKET_AUTH_TOKEN=<PACKET_API_KEY>
```

Instead of using placeholder text, we should use a concrete example:

```console
export PACKET_AUTH_TOKEN=xN3fd65gFHvkej875lkhDOP7UJ8746ds
```

We should trust that the users understand they need to paste their own key rather than the sample value.

https://github.com/kinvolk/lokomotive/blame/5e4f3c2884c1d018e29186d5aee68d4b51d1f653/docs/quickstarts/packet.md#L85-L86

>A [production-ready configuration](../../examples/packet-production) is already provided for ease of
use. Copy the example configuration to the working directory and modify accordingly.

- Rather than linking to a config file, we should paste its contents in the guide. DRY concerns can be solved using automation.
- I don't think we should try to create a production-ready cluster in a quickstart. We should opt for a basic, easy to deploy cluster if that means less steps and less things to worry about for the user.

**Comment by johananl** [comment]
Status:

- [x] Packet (PR open: #332)
- [ ] AWS
- [ ] AKS
- [ ] Bare metal

Once #332 is merged, I suggest we use the Packet quickstart as a "template" for the rest of the platforms.
